contracts/src/bridge/Bridge.sol

```diff
@@ -32,15 +32,23 @@ contract Bridge is OwnableUpgradeable, DelegateCallAware, IBridge {
     address[] public allowedInboxList;
     address[] public allowedOutboxList;
 
-    address public override activeOutbox;
+    address private _activeOutbox;
 
     /// @dev Accumulator for delayed inbox messages; tail represents hash of the current state; each element represents the inclusion of a new message.
     bytes32[] public override inboxAccs;
 
+    address private constant EMPTY_ACTIVEOUTBOX = address(type(uint160).max);
+
     function initialize() external initializer onlyDelegated {
+        _activeOutbox = EMPTY_ACTIVEOUTBOX;
         __Ownable_init();
     }
 
+    function activeOutbox() public view returns (address) {
+        if(_activeOutbox == EMPTY_ACTIVEOUTBOX) return address(uint160(0));
```

Comment:
we could add some comments to explain in the code (and [natspec](https://docs.soliditylang.org/en/v0.8.13/natspec-format.html#documentation-example)) 
```suggestion
    /// @dev returns the address of current active Outbox, or zero if no outbox is active
    function activeOutbox() public view returns (address) {
        // address zero is returned if no outbox is set, but the value used in storage
        // is non-zero to save users some gas. EIP-1153 would help here
        if(_activeOutbox == EMPTY_ACTIVEOUTBOX) return address(uint160(0));
```